ci/lambdas/github-event-handler/index.js

```diff
@@ -291,8 +291,12 @@ function triggerBuild(versionId, event, callback) {
 
     if (event.pull_request) {
         const num = event.pull_request.number;
+        const branch = event.pull_request.head.ref;
         environmentVariables.push({ name: 'PRX_GITHUB_PR', value: `${num}` });
+        environmentVariables.push({ name: 'PRX_BRANCH', value: branch});
     } else {
+        const branch = (event.ref || 'unknown').replace(/^refs\/heads\//, '');
```

Comment:
I haven't seen any - was mostly just covering all the bases here.  If we happen to add some other notification types down the road.